src/util/board.ts

```diff
@@ -0,0 +1,27 @@
+import { S3Client } from "@aws-sdk/client-s3";
+const path = require('path');
+const { Upload } = require("@aws-sdk/lib-storage");
+
+const client = new S3Client({
+  apiVersion: '2006-03-01',
```

Comment:
seeking to learn: Why do we have this apiVersion? Also why are there ! after the envvar names

Reply:
the ! part was to fix a type mismatch error. as for the apiVersion part, i'm not sure. @maxwn04 do you know why apiVersion is there?